module_oop/models.py

```diff
@@ -0,0 +1,100 @@
+import random
+import settings
+import exceptions
+
+
+class Enemy:
+    def __init__(self, level, health) -> None:
+        self.level = level
+        self.health = health
+
+    def decrease_health(self) -> None:
+        self.health -= 1
+        if self.health < 1: raise exceptions.EnemyDown
+
+
+    @staticmethod
+    def select_attack():
+        return random.randint(1, 3) # 1-Knight, 2-Thiev, 3-Wizart
+
+    @staticmethod
+    def select_defence() -> int:
+        return random.randint(1, 3)  # 1-Knight, 2-Thiev, 3-Wizart
+
+
+class Player:
+    def __init__(self, name, health, score, result_fight) -> None:
+        self.health = health
+        self.score = score
+        self.name = name
+        self.result_fight = result_fight
+
+
+    def attack(self, enemy: Enemy) -> None:
+        attack = self.select_attack()
+        defence = enemy.select_defence()
+        self.result_fight = self.fight(attack, defence)
+        print('result_fight: ', self.result_fight)
+        if self.result_fight == 'win':
+            enemy.decrease_health()
+            self.score += settings.SCORE_SUCCESS_ATTACK
+            print('YOUR ATTACK IS SUCCESSFUL!', 'enemy_health', enemy.health,
```

Comment:
![image](https://user-images.githubusercontent.com/22391544/233691004-a86169e5-a3e3-4d11-ae17-9cf83d55c3e0.png)
```suggestion
            print('YOUR ATTACK IS SUCCESSFUL!')
```
Чого вам так тягне дописати ще щось - я не розумію.